Gems/ScriptCanvas/Code/scriptcanvasgem_common_files.cmake

```diff
@@ -204,6 +200,10 @@ set(FILES
     Include/ScriptCanvas/Serialization/BehaviorContextObjectSerializer.cpp
     Include/ScriptCanvas/Serialization/DatumSerializer.cpp
     Include/ScriptCanvas/Serialization/RuntimeVariableSerializer.cpp
+    Include/ScriptCanvas/Data/BehaviorContextObject.cpp
+    Include/ScriptCanvas/Data/BehaviorContextObjectPtr.cpp
+    Include/ScriptCanvas/Data/Data.cpp
+    Include/ScriptCanvas/Data/DataRegistry.cpp
     Include/ScriptCanvas/Data/DataTrait.cpp
     Include/ScriptCanvas/Data/PropertyTraits.cpp
```

Comment:
Repeat: These include paths should be moved upwards to maintain alphabetical order